spec.bs

```diff
@@ -1346,13 +1353,14 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
     * To parse the value |result|:
       1. Set |auctionConfig|'s [=auction config/per buyer signals=] to a new [=ordered map=] whose

```

Comment:
so it always sets |auctionConfig|'s [=auction config/per buyer signals=] to a map, and not setting it to null when |result| is null. In our code, we have optional<map> for these fields ([example: per_buyer_currencies](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/public/common/interest_group/auction_config.h;drc=e66b343b5554785a32ad988bfe5c3c524f5e1857;bpv=1;bpt=1;l=185?gsn=per_buyer_currencies&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fchromium.googlesource.com%2Fcodesearch%2Fchromium%2Fsrc%2F%2Fmain%3Flang%3Dc%252B%252B%3Fpath%3Dthird_party%2Fblink%2Fpublic%2Fcommon%2Finterest_group%2Fauction_config.h%23puSypB3se86sqliHUynw1QaSf_VEGQsWD-MXmthMPoY)), but seems no much difference, right? (if so, seems currently in spec these map fields can never be null, where their [definitions](https://wicg.github.io/turtledove/#auction-config-per-buyer-cumulative-timeouts) say they could)

Reply:
I think I am just wrong here, and we do pass null through --- not sure why I misread code that way. Thanks!
(Will get back to this in a bit, need to finish other task first)

Reply:
Changed. Looks like deprecatedRenderURLReplacement doesn't do nullability, though.
The difference is whether we add the field when we serialize the config back to seller worklet or not.
So observable, but somewhat less important than what motivated this CL --- saying that resolving these promises with null or undefined (rather than empty objects) when you want nothing is acceptable.

Reply:
is it worthwhile to change deprecatedRenderURLReplacement's implementation?